index.html

```diff
@@ -1,13 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-   
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Onomeh's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500,600&amp;display=swap"
+        />
+      <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+        <header class="toolbar-header">
+        <div class="tools">
+          <a href="/" class="logo">Onomeh</a>
+            <button class="button" type="button">
+              <img src="../images/Icons/Union.png" alt="more" />
+            </button>
+        </div>
+      </header>
+    
+    <main class="intro">
```

Comment:
- [ ] Instead of using the `main` tag here, let's use the `section` tag here.
- **Requirement** The container of the section is a `section` tag.